relayer-cli/Cargo.toml

```diff
@@ -40,7 +40,7 @@ tokio = { version = "1.0", features = ["full"] }
 tracing = "0.1.34"
 tracing-subscriber = { version = "0.3.11", features = ["fmt", "env-filter", "json"]}
 eyre = "0.6.8"
-color-eyre = "0.6"
+color-eyre = "0.6.2"
```

Comment:
cargo update doesn't do the trick?

Reply:
It's just this crate that needed to be updated to fix conflicts with another dependency.